Update vm-memory (and keep its user linux-loader in sync) to match the version
packaged in Fedora, which was updated for vmm-sys-util's CVE-2023-50711.

--- a/src/utils/Cargo.toml
+++ b/src/utils/Cargo.toml
@@ -15,7 +15,7 @@
 thiserror = "1.0.32"
 displaydoc = "0.2.4"
 vmm-sys-util = "0.12.1"
-vm-memory = { version = "0.13.0", features = ["backend-mmap", "backend-bitmap"] }
+vm-memory = { version = "0.14.0", features = ["backend-mmap", "backend-bitmap"] }
 log-instrument = { path = "../log-instrument", optional = true }
 
 [dev-dependencies]
--- a/src/vmm/Cargo.toml
+++ b/src/vmm/Cargo.toml
@@ -18,7 +18,7 @@
 lazy_static = "1.4.0"
 libc = "0.2.117"
 memfd = "0.6.3"
-linux-loader = "0.10.0"
+linux-loader = "0.11.0"
 serde = { version = "1.0.136", features = ["derive", "rc"] }
 semver = { version = "1.0.17", features = ["serde"] }
 serde_json = "1.0.78"
@@ -29,7 +29,7 @@
 vhost = { version = "0.10.0", features = ["vhost-user-frontend"] }
 vm-allocator = "0.1.0"
 vm-superio = "0.7.0"
-vm-memory = { version = "0.13.1", features = ["backend-mmap", "backend-bitmap"] }
+vm-memory = { version = "0.14.0", features = ["backend-mmap", "backend-bitmap"] }
 log = { version = "0.4.17", features = ["std", "serde"] }
 aes-gcm =  { version = "0.10.1", default-features = false, features = ["aes"] }
 base64 = "0.21.0"
